scheduler/scheduler_test.go

```diff
@@ -21,27 +21,48 @@ import (
 	"testing"
 	"time"
 
+	"github.com/getgort/gort/command"
+	"github.com/getgort/gort/config"
 	"github.com/getgort/gort/data"
+	"github.com/getgort/gort/data/rest"
+	"github.com/getgort/gort/dataaccess"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCommandSchedulerFull(t *testing.T) {
-	scheduledCommands := StartScheduler()
+	// Init Gort
+	require.NoError(t, config.Initialize("../testing/config/no-database.yml"))
+
+	da, err := dataaccess.Get()
+	require.NoError(t, err)
 
+	require.NoError(t, da.Initialize(context.Background()))
```

Comment:
Can you just create and use one `context`, even if it is `context.Background()` ?